src/router/router.ts

```diff
@@ -49,6 +50,11 @@ export function handleSwapped(event: Swapped): void {
   routerSwapped.blockNumber = event.block.number;
   routerSwapped.time = event.block.timestamp;
   routerSwapped.save();
+
+  let routerExchange = createOrLoadRouterExchange(id);
```

Comment:
this function is for routerSwapped, why do you add routerExchange here? It must belong to the function `handleExchange` above, right?

Reply:
`handleSwapped`event but the event also contains information (`srcToken`, `dstToken`) exposed in `routerExchange` -> so I load routerExchange(id) and mutate it